test/e2e/specs/selenium.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const axios = require('axios');
+
+describe('wikipedia preview - ', () => {
+
+    it('should open article on another tab', async (done) => {
+        await browser.maximizeWindow()
+        await browser.url('https://wikimedia.github.io/wikipedia-preview')
+        el = await $('.content-en > p:nth-child(2) > span:nth-child(2)')
+        await el.moveTo()
+        await el.click()
+        el=await $('div.wp-text-content > div.wp-title')
```

Comment:
You should have spaces around = like on line 9. Also applies to lines 15 and 17.

Reply:
how can I use the linter for this?

Reply:
_we need the linter! please wait for few days until we include it in the master._

Reply:
fixed